docs/rollup.config.js

```diff
@@ -20,10 +20,10 @@ const mode = process.env.NODE_ENV;
 const dev = mode === 'development';
 const legacy = !!process.env.SAPPER_LEGACY_BUILD;
 
-const onwarn = (warning, onwarn) => (
-  warning.code === 'CIRCULAR_DEPENDENCY'
-  && /[/\\]@sapper[/\\]/.test(warning.message)
-) || onwarn(warning);
+const onwarn = (warning, onwarn) =>
+  (warning.code === 'MISSING_EXPORT' && /'preload'/.test(warning.message)) ||
```

Comment:
I kinda prefer keeping the operator in the beginning of the next line as opposed to the end of the current one. Easier to read, because by the time your eyes latch on to a new line, you'll forget what the operator was

Reply:
I took it directly from the template, and it helps in alignment, but fine.